app/components/page.tsx

```diff
@@ -20,11 +21,9 @@ const Components = () => {
         </Preview>
         <Preview
           name="Settings"
-          variants={["list", "grid"]}
-          codeStringGenerator={connCodeGen}
+          variants={[]}
+          codeStringGenerator={settingsCodeGen}
           description="lorem Lorem ipsum dolor sit amet consectetur adipisicing elit. Minima exercitationem rem quos, at consequuntur magni blanditiis quae libero debitis repudiandae dolorum dicta amet explicabo nihil? Id voluptas nihil culpa! Reiciendis."
-          themeVariants={false}
-          layoutVariants={false}
         >
```

Comment:
Why did you delete these?

Reply:
Because the settings doesn't make use of theme and layout prop